smart-contracts/wasm-chain-integration/src/utils.rs

```diff
@@ -64,10 +65,37 @@ impl<I> machine::Host<I> for TrapHost {
 }
 
 /// A host which traps for any function call apart from `report_error` which it
-/// prints to standard out.
-pub struct TestHost;
+/// prints to standard out and `get_random` that calls a random number
+/// generator.
+pub struct TestHost<R> {
+    /// A RNG for randomised testing.
+    rng:      Option<R>,
+    /// A flag set to `true` if the RNG was used.
+    rng_used: bool,
+}
 
-impl validate::ValidateImportExport for TestHost {
+impl TestHost<SmallRng> {
+    /// Create a new `TestHost` instance without a RNG instance
+    pub fn uninitialized() -> Self {
```

Comment:
Since we use this in benchmarks calling this function might cause a bit of a bias. We shoulud mark it as `const` or change how it is used in benchmarks.

Reply:
Done. Marked as `const`.